.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -42,3 +42,10 @@ Enter your extended release note in the block below. If the PR requires addition
 ```release-note
 
 ```
+<!--
+**Additional documentation, like links to enhancement proposals (KEP)**:
```

Comment:
Should this line be a part of the comment too?

Reply:
I think so, adding further documentation is not an optional step. :)

Reply:
The problem is, because it's inside the comment, nobody will see it. The comment is only for instructions.

Reply:
Alright, changed it that we only comment the actual instructions.